README.md

```diff
@@ -49,21 +48,29 @@ This data augmentation is motivated by the large inter-rater variability that we
 
 ## Installing
 
-This project requires Python 3.6 and PyTorch >= 1.2.0. We recommend you work under a virtual environment:
+
```

Comment:
On PyPi some images from the Readme doesn't display properly: 
![image](https://user-images.githubusercontent.com/49137243/81583222-0c9d8180-937f-11ea-869d-e3077f3322d1.png)
We should change the paths of these images.

Reply:
weird that the badges were absolute but the images were relative paths, now fixed :)